### PR TITLE
Benchmark v3

### PR DESCRIPTION
**Pull request by JosephDavidsonKSWH**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,15 +86,8 @@ The datasets that are implemented for this benchmark can be found in `./datasets
...
 colours
-conflicting_personal_info
-delayed_recall
-how_to_think
-instruction_recall
 jokes
-kv
-locations
 locations_directions
-names
```
Comment (dcasbol) [inline README.md:97]: These datasets might not be 100% reliable and therefore not included in the final configuration, but they are still implemented in the repository.

File: README.md [hunk 2]
```diff
@@ -118,33 +111,45 @@ The repository consists of four parts:
 More details for each of these parts can be found here: [datasets](datasets/README.md), [models](model_interfaces/README.md), [runner](runner/README.md), [reports](reporting/README.md).
 
 
-## Benchmark 2 - 03/2024
+## Benchmark 3 - 04/2024
```
Comment (dcasbol) [inline README.md:114]: The section "Configurations" seems outdated. Also, what do you think about placing the old configuration files under `configurations/older` or something like that?

File: model_interfaces/joe_ltm_prototype.py
Comment (dcasbol) [inline model_interfaces/joe_ltm_prototype.py:1]: Do you want to keep this empty file?

Comment (JosephDavidsonKSWH) [reply model_interfaces/joe_ltm_prototype.py:1]: No, it is getting removed.

File: README.md (hunk 1 above)
Comment (JosephDavidsonKSWH) [reply README.md:97]: I guess... I can restore it.

**Pull request merged**